server/routes.js

```diff
@@ -14,6 +14,18 @@ emailRewrite = require('./lib/email_rewrite.js'),
       secLog = require('./lib/security_logging'),
          url = require('url');
 
+var versionData;
+try {
+  versionData = require('../version.json');
+} catch(e) {
+  versionData = {
+    commit: "not found",
+    source: "not found",
+    version: "not found",
+    build: "not found"
+  };
+}
+
```

Comment:
@jrgm is this an idiomatic way of doing it? After writing so much golang I forgotten nodejs tricks ;)